{{cookiecutter.project_shortname}}/docker-services.yml

```diff
@@ -72,15 +72,18 @@ services:
       - "15672:15672"
       - "5672:5672"
   es:
-    {%- if cookiecutter.elasticsearch == '6'%}
+    {%- if cookiecutter.elasticsearch == '7' %}
+    image: docker.elastic.co/elasticsearch/elasticsearch-oss:7.2.0
+    {%- elif cookiecutter.elasticsearch == '6' %}
     image: docker.elastic.co/elasticsearch/elasticsearch-oss:6.6.0
-    {%- elif cookiecutter.elasticsearch == '5' %}
-    image: elasticsearch:5.6.4
     {%- endif %}
     restart: "always"
     environment:
       - bootstrap.memory_lock=true
       - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
+    {%- if cookiecutter.elasticsearch == '7' %}
+      - discovery.type=single-node
```

Comment:
Shouldn't that be for ES6 as well?

Reply:
We could add it for ES6 as well just to have fewer if-statements but it only seems to be needed for ES7.